### PR TITLE
crud: bump rocks minor version with compatibility fixes

### PR DESCRIPTION
**Pull request by CuriousGeorgiy**
Needed for tarantool/tarantool#8147

- [x] Tests (see [documentation](https://pkg.go.dev/testing) for a testing package)
- [x] Changelog (see [documentation](https://keepachangelog.com/en/1.0.0/) for changelog format)
- [x] Documentation (see [documentation](https://go.dev/blog/godoc) for documentation style guide)

Related issues:

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -81,9 +84,15 @@ jobs:
         run: |
           tt init
           sudo tt install tarantool master
+          
+      - name: Setup Tarantool from PR
+        if: matrix.tarantool == 'pr/8630'
+        run: |
+          tt init
+          tt install tarantool pr/8630
```
Comment (DifferentialOrange) [inline .github/workflows/testing.yml:92]: 2.x latest setup uses `sudo install`, you may need this too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Support `fetch_latest_metadata` option for crud requests with metadata (#335)
 - Support `noreturn` option for data change crud requests (#335)
 - Support `crud.schema` request (#336)
+- Support Tarantool tuples over IPROTO in crud (#337)
```
Comment (DifferentialOrange) [inline CHANGELOG.md:23]: go-tarantool does not depends on crud, the fix is related only to test infrastructure

File: .github/workflows/testing.yml (hunk 1 above)
Comment (CuriousGeorgiy) [reply .github/workflows/testing.yml:92]: The workflow succeeded, looks like `sudo` is unnecessary.

**Review by DifferentialOrange — APPROVED** [review]

**Review by oleg-jukovec — APPROVED** [review]

**Comment by oleg-jukovec** [comment]
Thank you for the patch!

**Pull request merged**